src/main/java/org/cyclonedx/CycloneDxSchema.java

```diff
@@ -73,16 +73,30 @@ public double getVersion() {
     }
 
     /**
-     * Returns the CycloneDX JSON Schema for the specified schema version.
+     * Returns the CycloneDX JsonSchema for the specified schema version.
      * @param schemaVersion The version to return the schema for
      * @param strict if true, the strict schema will be used which prohibits non-defined properties from being present in the BOM
      * @return a Schema
      * @throws IOException when errors are encountered
-     * @throws URISyntaxException when errors are encountered
-     * @since 3.0.0
+     * @since 5.0.0
      */
-    public org.everit.json.schema.Schema getJsonSchema(CycloneDxSchema.Version schemaVersion, boolean strict) throws IOException, URISyntaxException {
-        return getJsonSchema12(strict);
+    public JsonSchema getJsonSchema(CycloneDxSchema.Version schemaVersion, boolean strict, final ObjectMapper mapper)
```

Comment:
@stevespringett I would love to come up with some way to not return the `JsonSchema` itself, and just something else private/protected so if implementations change, you don't have to modify the API contract. I don't know what to do quite yet, but a lot of this work mostly had me thinking about that, ha.